test/map/test_core.py

```diff
@@ -36,29 +41,41 @@ def run(self, text: str, **kwargs: Any) -> Optional[FieldMap]:  # type: ignore
         return None
 
 
-@lambda_mapper
+class SquareHitTracker:
+    def __init__(self):
+        self.n_hits = 0
+
+    def __call__(self, x: float) -> float:
+        self.n_hits += 1
+        return x ** 2
+
+
+@lambda_mapper()
 def square(x: DataPoint) -> DataPoint:
     x.num_squared = x.num ** 2
     return x
 
 
-@lambda_mapper
+@lambda_mapper()
 def modify_in_place(x: DataPoint) -> DataPoint:
     x.d["my_key"] = 0
     x.d_new = x.d
     return x
 
 
 class TestMapperCore(unittest.TestCase):
-    def _get_x(self) -> SimpleNamespace:
-        return SimpleNamespace(num=8, text="Henry has fun")
+    def _get_x(self, num=8, text="Henry has fun") -> SimpleNamespace:
+        return SimpleNamespace(num=num, text=text)
 
     def _get_x_dict(self) -> SimpleNamespace:
         return SimpleNamespace(num=8, d=dict(my_key=1))
 
     def test_numeric_mapper(self) -> None:
         square.set_mode(MapperMode.NAMESPACE)
         x_mapped = square(self._get_x())
+        # NB: not using `self.assertIsNotNone` due to mypy
```

Comment:
Could we do `self.assertTrue(x_mapped is not None)` instead?

Reply:
Hmmm, I'll try it

Reply:
Answer: nope